pkg/autodiscovery/providers/kubelet.go

```diff
@@ -109,6 +122,17 @@ func parseKubeletPodlist(podlist []*kubelet.Pod) ([]integration.Config, error) {
 
 			configs = append(configs, c...)
 		}
+		adErrors = append(adErrors, utils.ValidateAnnotationsMatching(pod.Metadata.Annotations, containerIdentifiers, containerNames, adPrefix)...)
+		for _, err := range adErrors {
+			log.Errorf("Can't parse template for pod %s: %s", pod.Metadata.Name, err)
```

Comment:
I don't think we need to log the validation errors, they will be very verbose as we run this method every ten seconds. Printing them on the status would be enough imho!
```suggestion
```